packages/@react-stately/toast/test/useToastState.test.js

```diff
@@ -102,18 +103,26 @@ describe('useToastState', () => {
     expect(result.current.toasts.length).toEqual(0);
   });
 
-  it('should not call onRemove via onAdd when there is an actionLabel', async () => {
-    let timeoutToast = {
-      content: 'Action Toast',
-      props: {variant: 'info', timeout: 1, actionLabel: 'Undo'}
-    };
-    let {result} = renderHook(() => useToastState());
-    expect(result.current.toasts.length).toEqual(0);
-    act(() => result.current.onAdd(timeoutToast.content, timeoutToast.props));
-
-    jest.runAllTimers();
-
-    expect(result.current.toasts.length).toEqual(1);
-    expect(result.current.toasts[0].timer).toBe(undefined);
+  describe('timers', () => {
```

Comment:
fixed this because jest was spitting out a terrible warning
this has influenced me to make a separate PR for others https://github.com/adobe-private/react-spectrum-v3/pull/265